setup.py

```diff
@@ -0,0 +1,16 @@
+import os
+from setuptools import setup, find_packages
+
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
+
```

Comment:
You are right I realized it does not work with github repo links for example. I wasn't sure what would be a better way to do it.